stable/nginx-ingress/README.md

```diff
@@ -51,6 +51,8 @@ Parameter | Description | Default
 `controller.image.pullPolicy` | controller container image pull policy | `IfNotPresent`
 `controller.config` | nginx ConfigMap entries | none
 `controller.defaultBackendService` | default 404 backend service; required only if `defaultBackend.enabled = false` | `""`
+`controller.scope.enabled` | limit the scope of the ingress controller | `false` (watch all namespaces)
+`controller.scope.namespace` | namespace to watch for ingress | `""` (use the release namespace)
```

Comment:
My intention was to add the possibility to limit the scope of the ingress controller to the namespace of the helm chart release, without the need to specify the namespace twice.